File: 3731-find-missing-elements/3731-find-missing-elements.java
class Solution {
    public List<Integer> findMissingElements(int[] nums) {
        List<Integer> n = new ArrayList<>();
        for (int num : nums) {
            n.add(num);
        }
        
        Collections.sort(n);
        List<Integer> lst = new ArrayList<>();
        
        for (int i = n.get(0); i < n.get(n.size() - 1); i++) {
            if (!n.contains(i)) {
                lst.add(i);
            } else {
                continue;
            }
        }
        
        return lst;
    
    }
}

File: 3731-find-missing-elements/3731-find-missing-elements.py
class Solution:
    def findMissingElements(self, nums: List[int]) -> List[int]:
        n = sorted(nums)
        lst = []
        for i in range(n[0],n[-1]):
            if i not in n:
                lst.append(i)
            else:
                continue
        return lst